### PR TITLE
Feature/committees cosponsors ayesha

### PR DESCRIPTION
**Pull request by ayeshamk**

### CODE REVIEW

File: server_py/flatgov/flatgov/dev.py [hunk 1]
```diff
@@ -9,15 +9,15 @@
 
 
 DATABASES = {
```
Comment (aih) [inline server_py/flatgov/flatgov/dev.py:11]: We need to figure out what to do with dev.py. If you prefer to have sqlite as the dev backend, we should remove dev.py from git; I'm using postgresql.

**Review by aih — CHANGES_REQUESTED** [review]

**Comment by aih** [comment]
1. Leave `dev.py` as it was (with postgresql as default) or remove it from git (and put in a dev-sample.py).
2. Add the id for each committee after the committee name, so there is a key for users. For example, in the view below, a user may not know what `HSED` stands for. We can resolve this by putting `(HSED)` after the appropriate committee name in the list at the top. I do see the reason not to put the whole committee name in the column; it takes a lot of space and may require a reduction in font size.
image

**Comment by ayeshamk** [comment]
> 2. Add the id for each committee after the committee name, so there is a key for users. For example, in the view below, a user may not know what `HSED` stands for. We can resolve this by putting `(HSED)` after the appropriate committee name in the list at the top. I do see the reason not to put the whole committee name in the column; it takes a lot of space and may require a reduction in font size.

There is no full form given for the acronyms in either of the tables. I found a committee field in Bills table under committee dict. I will add that field for a given committee id. For example, "committee": "House Appropriations", "committee_id": "HSAP".

**Comment by aih** [comment]
Using the 'committee' field in the committees_dict, as you suggest, is
probably the best approach, and should still work into the future, if there
are new committees.

The committee codes appear to be the last part of the url path for
committees listed here:
https://www.congress.gov/help/field-values/current-committees

I was not able to find a straightforward table of committee name vs
committee_id.

**Review by aih — CHANGES_REQUESTED** [review]
The logic for adding committee information to a cosponsor only provides for one committee per cosponsor. A cosponsor may be on multiple committees, so the cosponsor should have a `committees` array (`cosponsor['committees']`), and this array should be of the form 1[{committee_id:'...', rank:'...'}, {committee_id:'...',rank:'...'}...]

**Comment by aih** [comment]
In today's call, we (Daniel) decided:

1. Change 'On Assigned Committee' to 'Current Assigned Committee' (I'll create a separate issue for this).
2. We will show a comma-separated list of the full committee name in the 'Current Assigned Committee' column.
3. The 'rank' will be '1' if the cosponsor is rank 1 on *any* committee. Otherwise, the rank is the rank of the cosponsor on the first committee in the list (there is no principled reason to chose one committee over another given our data).
4. We will consider adding a column showing whether a cosponsor is a current member of Congress; the Cosponsors db table contains only current members, so if the bioguide id of the cosponsor is not in the Cosponsors table, the cosponsor is not currently in Congress.

**Comment by ayeshamk** [comment]
Per discussion with Daniel, there are additional requirements. I have created issues and will be working on them.

**Pull request closed without merging**